thoth/python/source.py

```diff
@@ -382,6 +490,7 @@ def get_package_hashes(self, package_name: str, package_version: str, with_inclu
             return self._warehouse_get_package_hashes(package_name, package_version, with_included_files)
 
         artifacts_sha = self._download_artifacts_sha(package_name, package_version, with_included_files)
+        artifacts_syms = self._download_artifacts_syms(package_name, package_version)
```

Comment:
It would be good to rework this slightly - pass also a path to an artifact. This way we are downloading artifact multiple times - this can be network hungry especially given the size some of the wheel files have - e.g. tensorflow-gpu has 350Mi, this way we will download it multiple times in our analyzers - each time we compute digests, symbols, ...